sdk/core/src/main/java/com/klaviyo/core/config/Log.kt

```diff
@@ -2,6 +2,34 @@ package com.klaviyo.core.config
 
 interface Log {
 
+    /**
+     * Log
+     */
+    enum class Level {
+        None,
+        Verbose,
+        Debug,
+        Info,
+        Warning,
+        Error,
+        Assert
+    }
+
+    /**
+     * The minimum log level to output
+     *
+     * To configure from manifest, specify the minimum log level to output:
+     *
+     * 0 = Disable logging
```

Comment:
Hmm...is this reversed from the setting we documented?

Reply:
Oh crud, yeah good catch, now to remember which one is correct 🤣

Reply:
Maybe not reversed is assert in the wrong spot though or doesn't matter?

Reply:
I think the readme is currently incorrect. I think I went with 0 = disable, 1 = verbose+, 2 = debug+ etc.

Reply:
https://developer.android.com/reference/android/util/Log it should match stuff in here right?

Reply:
Yes, in spirit anyway. For some reason theirs go from 2 to 7. But lower number = lower severity was my intent. I fixed values in the readme